drivers/lora/sx126x_common.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019 Manivannan Sadhasivam
+ * Copyright (c) 2020 Andreas Sandberg
+ * Copyright (c) 2021 Fabio Baltieri
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
This file should presumably needs include guards.

Reply:
Oh yeah, added now.